sections/04_study.md

```diff
@@ -709,101 +709,92 @@ the next few years.
-accomplish both tasks in one network? Does functional/taxonomic classification need to be separate processes?  The largest
-potential in deep learning is to learn "everything" in one complex network, with a plethora of labeled (reference) data and
-unlabeled (microbiome experiments) examples.
+Metagenomics, which refers to the study of genetic material, 16S rRNA and/or
+whole-genome shotgun DNA from microbial communities, has revolutionized the
+study of micro-scale ecosystems within us and around us. In recent years,
+machine learning has proved to be a powerful tool for metagenomic analysis. 16S
+rRNA has long been used to deconvolve mixtures of microbial genomes, yet this
+ignores >99% of all reads. Subsequent tools aimed to classify 300-3000bp reads
+from complex mixtures of microbial genomes based on tetranucleotide frequencies
+(which are characteristic for different organisms [@tag:Karlin]) using
+supervised [@tag:McHardy @tag:nbc] or unsupervised methods such as
+self-organizing maps (SOM) [@tag:Abe]. Then, researchers began to use techniques
+that could estimate relative abundances from an entire sample, which is much
+faster than classifying individual reads [@tag:Metaphlan @tag:wgsquikr
+@tag:lmat]. There is also great interest in identifying and annotating sequence
+reads [@tag:yok @tag:Soueidan]. However, the focus on taxonomic/functional
+annotation is just the first step. Several groups have proposed methods to
+determine host or environment phenotypes from the organisms that are identified
+[@tag:Guetterman @tag:Knights @tag:Stratnikov @tag:Segata] or overall sequence
+compsition [@tag:Ding]. Also, researchers have looked into how feature selection
+can improve classification [@tag:Liu @tag:Segata], and techniques have been
+proposed that are classifier-independent [@tag:Ditzler @tag:Ditzler2].
+
+How have neural networks (NNs) been of use? Most neural networks are being used
+for phylogenetic classification or functional annotation from sequence data,
+where there is a lot of data for training (and thus suitable for NNs). Neural
+networks have been applied successfully to gene annotation (e.g. Orphelia
+[@tag:Hoff] and FragGeneScan [@doi:10.1093/nar/gkq747]), which usually has
+plenty of training examples.  Representations (similar to Word2Vec
+[@tag:Word2Vec] in natural language processing) for protein family
+classification has been introduced and classified with a skip-gram neural
+network [@tag:Asgari]. Recurrent neural networks show good performance for
+homology and protein family identification [@tag:Hochreiter @tag:Sonderby].
+Interestingly, Hochreiter, who invented Long Short Term Memory (LSTM), delved
+into homology/protein family classification in 2007, and therefore, deep
+learning is deeply rooted in functional classification methods.
+
+One of the first techniques of “de novo” genome binning used self-organizing
+maps, a type of NN [@tag:Abe]. Essinger et al. use ART, a neural network
```

Comment:
This `Abe` reference on SOMs feel redundant with the reference above to the same paper.